GithubApp/CustomeCells/SearchTableViewCell.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  SearchTableViewCell.swift
+//  GithubApp
+//
+//  Created by BrainX on 07/06/2022.
+//
+
+import UIKit
+
+class SearchTableViewCell: UITableViewCell {
+
+    // MARK: - IB Outlets
+    @IBOutlet weak var outerView: UIView!
+    @IBOutlet weak var languageLabel: UILabel!
+    @IBOutlet weak var personImage: UIImageView!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var backgroungImage: UIImageView!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        self.layer.cornerRadius = 30
+        nameLabel.font = UIFont(name: "Gilroy", size: 17)
+        configureOuterView()
+        configurePersonImage()
+        configureLanguageLabel()
+    }
+    
+    // MARK: - Functions For Configurations
+    func configureOuterView(){
+        outerView.layer.cornerRadius = 20
+        outerView.layer.borderWidth = 1
+        outerView.layer.borderColor = UIColor.lightGray.cgColor
+    }
+
+    func configureDescriptionLabel(){
+        descriptionLabel.font = UIFont(name: "Gilroy", size: 17)
+    }
+
+    func configureNameLabel(){
+        nameLabel.font = UIFont(name: "Gilroy", size: 17)
+    }
+
```

Comment:
you dont have to create a separate method for each veiw/label. you can group some configurations for all views like
`setupFonts`. You can also use one single method for all configurations like `configureViews` etc.